lib/logstash/inputs/LogStashRecordProcessor.rb

```diff
@@ -0,0 +1,68 @@
+# encoding: utf-8
+#
+#Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#
+require "java"
+
+require "logstash-input-dynamodb_jars"
+java_import "com.amazonaws.services.kinesis.clientlibrary.types.ShutdownReason"
+java_import "java.lang.IllegalStateException"
+java_import "org.apache.log4j.LogManager"
+
+module Logstash
+  module Inputs
+    module DynamoDB
+      class LogStashRecordProcessor
```

Comment:
The implementation of these classes follows a convention that has been set when using KCL.  I believe even if it's redundant, it's better to keep a consistent experience across our KCL applications.